src/Model/Locale.php

```diff
@@ -409,7 +408,7 @@ public static function getByLocale($locale): ?Locale
         }
 
         // Get filtered locale
-        return Locale::getCached()->find('Locale', $locale);
+        return Locale::get()->setUseCache(true)->find('Locale', $locale);
```

Comment:
`getCached()` cached a bunch of records in an `ArrayList`.
Calling `find()` on that list requires filtering through _the full list of locales_ every time this is called, even though we just want the same record every time.

Caching this record separately ends up being more efficient and was the main driver for finally adding this level of caching to `DataList` in the first place.